src/uct/ib/base/ib_md.c

```diff
@@ -309,6 +311,29 @@ uint8_t uct_ib_md_get_atomic_mr_id(uct_ib_md_t *md)
 #endif
 }
 
+static void uct_ib_md_print_mem_reg_err_msg(ucs_log_level_t level, void *address,
+                                            size_t length, uint64_t exp_access,
+                                            const char *exp_prefix)
+{
+    char msg[200] = {0};
+    struct rlimit limit_info;
+
+    ucs_snprintf_zero(msg, sizeof(msg),
+                      "ibv_%sreg_mr(address=%p, length=%zu, %saccess=0x%lx) failed: %m. ",
+                      exp_prefix, address, length, exp_prefix, exp_access);
+
+    /* Check the value of the max locked memory which is set on the system
+     * (ulimit -l) */
+    if ((!getrlimit(RLIMIT_MEMLOCK, &limit_info)) &&
+        (limit_info.rlim_cur != RLIM_INFINITY)) {
+        ucs_snprintf_zero(msg + strlen(msg), sizeof(msg) - strlen(msg),
+                          "Please set max locked memory (ulimit -l) to 'unlimited' "
```

Comment:
should move the `". "` from the end of line 322 to the beginning of the string in this line